recipes/default.rb

```diff
@@ -0,0 +1,71 @@
+#
+# Cookbook:: bluematador_agent
+# Recipe:: default
+#
+# Copyright:: 2018, Blue Matador Inc., All Rights Reserved.
+
+case node[:platform]
+when 'debian', 'ubuntu'
+	apt_update 'update' do
+		action :update
+	end
+	package 'apt-transport-https' do
+		action :install
+	end
+
+	apt_repository 'bluematador-agent' do
```

Comment:
you never `apt-get update` on this, do you? doesn't seem like it would work the first time.

Reply:
apt_repository does an apt-get update